CMakeLists.txt

```diff
@@ -20,7 +21,28 @@ add_definitions(-DSQLITE_OMIT_LOAD_EXTENSION        # This one's for SQLiteCpp
                 -DUSE_IPV6                          # Enable IPv6 support in CivetWeb
                 -DCMAKE                             # Let the source know this is a CMAKE build
                 -D__STDC_LIMIT_MACROS               # Ensure the presence of UINT64_MAX, etc
-                -DNOMINMAX)                         # Get rid of pesky Windows macros for min and max
+                -DNOMINMAX                          # Get rid of pesky Windows macros for min and max
+                -DSQLITE_DEFAULT_WAL_SYNCHRONOUS=1
```

Comment:
These flags were removed here because SQLite is now built by SQLiteCpp's nested CMakeLists, right?
Did you modify that other file to include the same flags as here?